docs/hiring-guide/community-spotlight/interview-lana-brindley.rst

```diff
@@ -0,0 +1,38 @@
+
+**Can you talk a bit about what it's like to lead a technical writing team?**
+
+So leading a team is difficult but I find it very personally satisfying. It's certainly not for everyone, though. I think the most important thing is what I refer to as the difference between management and leadership. I wrote fairly extensively about this after I stood down as the OpenStack PTL (program technical lead) in 2017, because I found the difference between managing a corporate team and leading a community of volunteers to be quite stark. In a corporate sense, the people on your team are there because they really have to be: you are responsible for their livelihoods, so they do what you tell them to do. In a corporate environment, if you get a bad manager (and we all have, let's face it), you either have to find a way to move out of that team, or you have to put up with it. Community teams are completely different: bad leaders simply don't have any followers. In a community environment, you have no way of enticing or rewarding people for doing hard work, but the hard work still needs to be done, so you have to rely on charisma, persuasion, and handing out the occasional sticker or T-shirt. It was therefore very interesting to me to be able to look at those periods with hindsight while I was studying the MBA, and be able to draw the lines between management theory and why certain things I tried as both a manager and a leader either did or didn't work.
+
+**What advice you might have for new writers trying to break into the field, especially for those looking to contribute to open source?**
+
+For new writers starting out, my best advice would be to just start writing. You need to have something to show prospective employers, but you can create that in any number of ways. If you don't have anything from your current work, then go and install some bit of software and write about that, even if it's been written about a thousand times before. Also, remember that for a writer your resume is a writing sample. Make sure that you show that you can write prose as well as dot points, and be really mindful of the words you choose, and the layout and design.
+
+As for contributing to open source, I know that documentation is often held up as a good way to get started, but I can appreciate that it's not always easy. That said, most larger open source groups (especially those that are backed by organisations like SUSE and Red Hat) have some kind of mentoring program that can help you with your first contribution if you really don't know where to start. And that's probably my other big tip: no matter where you are in your career, always choose at least one mentor and one mentee. There is always something you can learn, and something you can teach
+
+Lana goes by ''loquacity'' on the Write the Docs Slack group, and @Loquacities on Twitter. You can find her in the ''#australia'', ''#managing-writers'', and ''#remote channels''.
```

Comment:
These double quotes need to be double back ticks

```rst
    ``this``
    ''not this''
```

Reply:
Fixed, thank you.